components/CodeSnippet.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="d-inline-block">
+    <div v-if="parameters">
+      <CTooltip content="Generate code snippet" placement="top">
+        <template #toggler="{ togglerId, on }">
+          <CButton
+            id="btn-code-snippet"
+            color="light"
+            :aria-describedby="togglerId"
+            class="btn-scenario-header"
+            v-on="on"
+            @click="() => { modalVisible = true; }"
+          >
+            <CIcon icon="cilCode" size="lg" class="text-secondary" />
+          </CButton>
+        </template>
+      </CTooltip>
+    </div>
+    <CModal
+      :visible="modalVisible"
+      aria-labelledby="modalTitle"
+      @close="() => { modalVisible = false }"
+    >
+      <CModalHeader>
+        <CModalTitle id="modalTitle">
+          Daedalus Code Snippet
```

Comment:
```suggestion
          DAEDALUS code snippet
```
The titlecase looks quite odd.